sdcm/nemesis.py

```diff
@@ -251,20 +263,56 @@ def disrupt_restart_with_resharding(self):
         self.log.info('Set back murmur3_partitioner_ignore_msb_bits value to 12')
         self.target_node.restart_node_with_resharding()
 
-    def _destroy_data(self):
-        # Send the script used to corrupt the DB
-        break_scylla = get_data_dir_path('break_scylla.sh')
-        self.target_node.remoter.send_files(break_scylla,
-                                            "/tmp/break_scylla.sh")
+    @retrying(n=10, allowed_exceptions=(NoKeyspaceFound, NoFilesFoundToDestroy))
+    def _choose_file_for_destroy(self, ks_cfs):
+        file_for_destroy = ''
+
+        ks_cf_for_destroy = random.choice(ks_cfs)  # expected value as: 'keyspace1.standard1'
+
+        ks_cf_for_destroy = ks_cf_for_destroy.replace('.', '/')
+        files = self.target_node.remoter.run("sudo find /var/lib/scylla/data/%s-* -type f" % ks_cf_for_destroy)
+        if files.stderr:
+            raise NoFilesFoundToDestroy('Failed to get data files for destroy in {}. Error: {}'.format(ks_cf_for_destroy,
+                                                                                                       files.stderr))
+
+        for file in files.stdout.split():
+            if not file or '/' not in file:
+                continue
+            file_path_splitted = file.split('/')
+            # The file name like: /var/lib/scylla/data/scylla_bench/test-f60e4f30c98f11e98d46000000000002/mc-220-big-Data.db
+            # For corruption we need to remove all files that their names are started from "mc-220-"
+            file_name = '-'.join(file_path_splitted[-1].split('-')[:2]) + '-*'
```

Comment:
This split won't work correctly for older versions than mc format.
Example file name for 'ka':
system-truncated-ka-7-Data.db
Example name of mc:
mc-43-big-Data.db

That's why the old shell script worked for ka, la and doesn't work for mc.
The cut -d '-' -f 1,2,3,4,5 should have changed to 1,2,3.

we should support both, maybe by splitting after the "digits" and "-"